exec/java-exec/src/test/java/org/apache/drill/exec/impersonation/TestImpersonationQueries.java

```diff
@@ -179,6 +199,15 @@ private static void createView(final String viewOwner, final String viewGroup, f
     assertEquals(viewPerms, status.getPermission().toShort());
   }
 
+  private static void createView(final String viewOwner, final String viewGroup, final String viewName,
```

Comment:
Any reason why (protected version of) this method cannot be moved to BaseTestImpersonation?